ide/jbuild

```diff
@@ -0,0 +1,20 @@
+(jbuild_version 1)
+
+(library
+ ((name ide_slave)
+  (flags -rectypes)
```

Comment:
You might want to consider using something like `(flags (:standard -rectypes))` which will append to the standard list of flags rather than overwriting them.

The :standard list of flags will be customizable soon enough as well (once profiles are added).

Reply:
Thanks! Indeed, I think I'll wait for profiles as `-rectypes` is a global flag for the Coq project so it makes sense to add it there and remove all `flags` declarations.

Reply:
Another way to fake having global flags is to have `flags.sexp` file in your project root, and then use `(:include ${ROOT}/flags.sexp)` in the flags field everywhere. But yeah, wait for profiles if you can. It will make this stuff obsolete.